src/lib/ngsi10/UpdateContextSubscriptionRequest.h

```diff
@@ -35,28 +35,22 @@
 #include "ngsi/SubscriptionId.h"
 #include "ngsi/NotifyConditionVector.h"
 #include "ngsi/Throttling.h"
-
+#include "ngsi10/SubscribeContextRequest.h"
 
 
 /* ****************************************************************************
 *
 * UpdateContextSubscriptionRequest - 
 */
-typedef struct UpdateContextSubscriptionRequest
+struct UpdateContextSubscriptionRequest : public SubscribeContextRequest
 {
-  Duration                       duration;               // Optional
```

Comment:
I understand that all the removed fields (duration, restriction, notifyConditionVector, throttling, restrictions) are got from the base clase. Is that correct?